src/main/java/it/gov/pagopa/pu/pagopapayments/endpoint/PaForNodeEndpoint.java

```diff
@@ -111,7 +117,30 @@ public PaSendRTV2Response paSendRTV2(@RequestPayload PaSendRTV2Request request)
     try {
       log.info("processing paSendRTV2 idPA[{}] notice[{}/{}]", request.getIdPA(), request.getReceipt().getFiscalCode(), request.getReceipt().getNoticeNumber());
       PaSendRtDTO paSendRtDTO = paSendRTMapper.paSendRtV2Request2PaSendRtDTO(request);
-      receiptService.processReceivedReceipt(paSendRtDTO);
+      var registryLoggerResult = registryLogger.execute(
+        paSendRtDTO.getFiscalCode(),
+        paSendRtDTO.getIdBrokerPA(),
+        request.getReceipt().getIdPSP(),
+        request.getReceipt().getIdChannel(),
+        request.getReceipt().getPaymentMethod(),
+        null, // @TODO: no ccp in request
```

Comment:
we could check on MyPay how these info are actually filled:
https://github.com/pagopa/p4pa-payhub-be/blob/develop/mypay4-be/src/main/java/it/regioneveneto/mygov/payment/mypay4/ws/server/fesp/PagamentiTelematiciCCP36Endpoint.java#L163

```suggestion
        request.getReceipt().getReceiptId(),
```